src/igr-tests/meson.build

```diff
@@ -1,33 +1,31 @@
 # Included from top-level meson.build
 
-## Prepare integration tests
-run_command([
-    'cp',
-    '-r',
-    './',
-    meson.current_build_dir()
-    ],
-    check : true
-)
+igr_tests_env += [
+     'IGR_FUNCTIONS=@0@/igr_functions'.format(meson.current_source_dir()),
+     'TEMP=@0@'.format(meson.current_build_dir()),
```

Comment:
Please call the output directory something more specific than TEMP. "IGR_OUTPUT" would be a good choice.